docs/api/topics/latecustomization.rst

```diff
@@ -0,0 +1,74 @@
+.. _latecustomization:
+.. versionadded:: 2
+
+====================
+ Late Customization
+====================
+
+Late customization allows loading of apps during the first-time experience on
+devices.
+
+.. http:get:: /api/v2/latecustomization/?carrier=(str:carrier)&region=(str:region)
+
+  Returns a list of late customization items for this carrier/region.
+
+  **Response**
+
+  :param latecustomization_id: The id for this late-customization item.
+  :type latecustomization_id: int
+  :param latecustomization_type: Either 'webapp' or 'extension'.
+  :type latecustomization_type: str
+
+  Includes the fields for either webapps or extensions, based on the item type.
+
+  :status 200: Successfully completed.
+
+
+  .. code-block:: json
+
+      {
+          "objects": [
+              {
+                  "latecustomization_type": "webapp",
+                  "latecustomization_id": 3,
+                  "id": 47911,
+                  "slug": "carrier-provided-app-1",
+                  ...
+              }
+              ...
+          ]
+      }
+
+
+.. http:post:: /api/v2/latecustomization/
+
+  Set the late-customization list for a given region and carrier. Replaces
```

Comment:
This just creates a single item doesn't it? I don't see how it could replace the existing list. The management portion is currently setup to manage individual items.